### PR TITLE
Update addon details box for channels

### PR DESCRIPTION
**Pull request by eviljeff**
my best guess at #3471 until Product/UX decide.

### CODE REVIEW

**Comment by eviljeff** [comment]
Listed versions only
screenshot 2016-11-15 14 10 31
Unlisted versions only
screenshot 2016-11-15 14 10 15
Mixed Listed and Unlisted
screenshot 2016-11-15 14 10 46
(latest unlisted version is blank because the file is disabled... and I thought it looked weird to have no details)

**Comment by eviljeff** [comment]
My Add-ons list in devhub (uses the same template)
screenshot 2016-11-15 15 53 49

File: src/olympia/devhub/templates/devhub/includes/addon_details.html [hunk 1]
```diff
@@ -99,24 +128,10 @@
...
-      <li class="queue-position" title="{{ _('Queues are not reviewed strictly in order') }}">
-        <strong>{{ _('Queue Position:') }}</strong>
-        {% trans position=position.pos|numberfmt,
-                 total=position.total|numberfmt %}
-          {{ position }} of {{ total }}
-        {% endtrans %}
-      </li>
-    {% endif %}
-  {% endwith %}
+{% if addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
```
Comment (diox) [inline src/olympia/devhub/templates/devhub/includes/addon_details.html]: Technically you can cheat and use `version.compatible_apps` with the right version. But `addon.compatible_apps` definitely relies on `current_version`.

`feature_compatibility` currently depend on the latest upload yeah.

Comment (diox) [inline src/olympia/devhub/templates/devhub/includes/addon_details.html]: Note: there is a slight difference in the way `Version` and `Addon` handle `compatible_apps`, when it comes to search providers and Personas. So maybe don't shave that yak yet...

File: src/olympia/devhub/templates/devhub/includes/addon_details.html [hunk 2]
```diff
@@ -15,76 +14,106 @@
...
-  <strong>{{ _('Visibility:') }}</strong>
-  {% if not addon.is_disabled and addon.is_listed %}
-    {{ _('Listed') }}
-  {% elif addon.is_disabled and addon.is_listed %}
-    {{ _('Hidden') }}
-  {% elif not addon.is_listed %}
-    {{ _('Unlisted') }}
-  {% endif %}
-</li>
+    {% set latest_unlisted_version = addon.find_latest_version(channel=amo.RELEASE_CHANNEL_UNLISTED) %}
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/includes/addon_details.html]: https://github.com/mozilla/addons-server/pull/4005/commits/47e2d4ff2a380d2a60136025cafd0506b5500427

**Pull request merged**